Cannot assign to "mustTesselate" because it is a constant (please upgrade in npm)

@babylonjs/core": ^5.22.0

X [ERROR] Cannot assign to "mustTesselate" because it is a constant 
`(mustTesselate as any) = null;` is fixed in github 👍 

Please update edgeRenderer.js in npm

Thanks

--- a/packages/dev/core/src/Rendering/edgesRenderer.ts
+++ b/packages/dev/core/src/Rendering/edgesRenderer.ts
@@ -678,7 +678,7 @@ export class EdgesRenderer implements IEdgesRenderer {
                 this._tessellateTriangle(triangle.edgesPoints, triangle.index, indices, remapVertexIndices);
             }
 
-            (mustTesselate as any) = null;
+            mustTesselate.length = 0;
         }
 
         /**

